src/alpakatest/AlpakaCore/alpakaConfig.h

```diff
@@ -5,21 +5,27 @@
 
 namespace alpaka_common {
   using Dim = alpaka::dim::DimInt<1u>;
+  using Dim2 = alpaka::dim::DimInt<2u>;
   using Idx = uint32_t;
   using Extent = uint32_t;
   using DevHost = alpaka::dev::DevCpu;
   using PltfHost = alpaka::pltf::Pltf<DevHost>;
   using WorkDiv = alpaka::workdiv::WorkDivMembers<Dim, Idx>;
+  using WorkDiv2 = alpaka::workdiv::WorkDivMembers<Dim2, Idx>;
   using Vec = alpaka::vec::Vec<Dim, Idx>;
+  using Vec2 = alpaka::vec::Vec<Dim2, Idx>;
 }  // namespace alpaka_common
 
 #ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
 #define ALPAKA_ACC_GPU_CUDA_ASYNC_BACKEND
 namespace alpaka_cuda_async {
   using namespace alpaka_common;
   using Acc = alpaka::acc::AccGpuCudaRt<Dim, Extent>;
+  using Acc2 = alpaka::acc::AccGpuCudaRt<Dim2, Extent>;
   using DevAcc = alpaka::dev::Dev<Acc>;
+  using DevAcc2 = alpaka::dev::Dev<Acc2>;
   using PltfAcc = alpaka::pltf::Pltf<DevAcc>;
+  using PltfAcc2 = alpaka::pltf::Pltf<DevAcc2>;
```

Comment:
Haha yes it does. Hmm but then what would you suggest, just templating it?

Reply:
I'd probably continue as is for now and see how it goes, and try to think simultaneously if and how this could be simplified. I'm not sure if additional templating would help much.

Reply:
Ok. Yes my feeling was that templating would make things heavier in caller code, while having a few more lines in Config does not harm too much.

Reply:
This is a long standing issue with Alpaka: the number of dimensions ends up giving different accelerators, which (IIRC) in turn require having different devices, etc.

I tried discussing this with the Alpaka team but without much luck :-(

We could just give up on it for the time being, and just use 3-dimensional indices all the time ?
At least under the hood (like CUDA and oneAPU do in practice) and assume that people will use the right number of indices when writing/calling kernels ?

Reply:
Thanks Andrea for the history and a suggestion. I opened an issue about this topic in #144 and suggest we move the further discussion there.